draft-ietf-quic-recovery.md

```diff
@@ -423,7 +423,7 @@ QUIC SHOULD delay sending acknowledgements in response to packets,
 but MUST NOT excessively delay acknowledgements of packets containing
 non-ack frames.  Specifically, implementaions MUST attempt to
 enforce a maximum ack delay to avoid causing the peer spurious
-timeouts.  The default maximum ack delay in QUIC is 25ms.
+timeouts.  The RECOMMENDED maximum ack delay in QUIC is 25ms.
```

Comment:
Why is only this recommendation capitalized? I think we should use capitalized RECOMMEND everywhere below

Reply:
Done.